src/FSharpPlus/Control/Traversable.fs

```diff
@@ -38,16 +38,6 @@ type Traverse =
 
     static member inline Traverse (t: Id<_>, f, [<Optional>]_output: 'R, [<Optional>]_impl: Default3) = Map.Invoke Id.create (f (Id.run t))
 
-    static member inline Traverse (t: _ seq, f, [<Optional>]_output: 'R, [<Optional>]_impl: Default3) =
-       let cons x y = seq {yield x; yield! y}
-       let cons_f x ys = Map.Invoke (cons: 'a->seq<_>->seq<_>) (f x) <*> ys
-       Seq.foldBack cons_f t (result Seq.empty)
-
-    static member inline Traverse (t: _ NonEmptySeq, f, [<Optional>]_output: 'R, [<Optional>]_impl: Default3) =
-       let cons x y = seq {yield x; yield! y}
-       let cons_f x ys = Map.Invoke (cons: 'a->seq<_>->seq<_>) (f x) <*> ys
-       Map.Invoke NonEmptySeq.ofSeq (Seq.foldBack cons_f t (result Seq.empty))
-
```

Comment:
Here's thing that might be causing the issue.

Try not changing the overloads and their signature and there should be no reason to change the overload resolution logic in Fable.

If you have to repeat code, go ahead.